src/client/a2/main/grid/layers/pixi-effect-object.ts

```diff
@@ -0,0 +1,8 @@
+import * as PIXI from 'pixi.js'
+
+export class PixiEffectObject {
```

Comment:
Shouldn't be here. I was anticipating we might have different effects that have common behaviors or attributes and this could be the base class.